apps/judicial-system/message-handler/src/app/messageHandler.service.ts

```diff
@@ -10,8 +10,10 @@ import {
   DefendantMessage,
   MessageService,
   MessageType,
+  OldMessageType,
```

Comment:
All code handling old messages can safely be removed once old messages have been flushed from the queue.